exporter/views.py

```diff
@@ -67,17 +67,31 @@ def do_bagit_export(assets, export_base_dir, export_filename_base):
 
         asset_id = asset_id.replace(":", "/")
         asset_path, asset_filename = os.path.split(asset_id)
+        item_path, item_filename = os.path.split(asset_path)
 
-        dest_path = os.path.join(export_base_dir, asset_path)
-        os.makedirs(dest_path, exist_ok=True)
+        asset_dest_path = os.path.join(export_base_dir, asset_path)
+        os.makedirs(asset_dest_path, exist_ok=True)
 
         # Build transcription output text file
-        text_output_path = os.path.join(dest_path, "%s.txt" % asset_filename)
-        with open(text_output_path, "w") as f:
+        asset_text_output_path = os.path.join(
+            asset_dest_path, "%s.txt" % asset_filename
+        )
+        with open(asset_text_output_path, "w") as f:
+            f.write(asset.latest_transcription or "")
+
+        item_text_output_path = os.path.join(asset_dest_path, "%s.txt" % item_filename)
+        with open(item_text_output_path, "a") as f:
             f.write(asset.latest_transcription or "")
-            if hasattr(settings, "ATTRIBUTION_TEXT"):
-                f.write("\n\n")
-                f.write(settings.ATTRIBUTION_TEXT)
+            f.write("\n\n")
+
+    # Add attributions to the end of all files found under asset_dest_path
+    if hasattr(settings, "ATTRIBUTION_TEXT"):
+        for dirpath, dirnames, filenames in os.walk(export_base_dir, topdown=False):
+            for each_text_file in filenames:
```

Comment:
Should this be filtered to avoid potential conflicts in the future if we ever start adding something which is not a text file?

```suggestion
            for each_text_file in (i for i in filenames if i.endswith(".txt")):
```

Reply:
Yes